homeassistant/components/acer_projector/switch.py

```diff
@@ -140,11 +140,11 @@ def update(self) -> None:
         else:
             self._available = False
 
-        for key in self._attr_attributes:
+        for key in self._attr_extra_state_attributes:
```

Comment:
This fails mypy becasue the definition of it is:
```python
_attr_extra_state_attributes: Mapping[str, Any] | None = None
```
so mypy thinks it can be `None`, to avoid this you can add above the for:
```python
if self._attr_extra_state_attributes is None:
    return
```